rust-runtime/aws-smithy-json/README.md

```diff
@@ -0,0 +1,3 @@
+# aws-smithy-json
+
+JSON serialization and deserialization primitives for Smithy-based services.
```

Comment:
Suggesting
```suggestion
JSON serialization and deserialization primitives for smithy-rs generated services.
```
To reduce confusion with https://crates.io/crates/smithy

It looks like other readmes also say "Smithy-based"